cypress/objects/timeLoggingPage.js

```diff
@@ -0,0 +1,38 @@
+class TimeLoggingPage {
+
+    
+    getPageTitle(){
+        return cy.get('.page__title')
+    }
+
+    calendar(){
+        return cy.get('.calendar')
+    }
+
+    titleForm(){
+        return cy.get('.tile.form')
+    }
+
+    infoTitle(){
+        return cy.get('.user-info__title')
+    }
+
+    mainNav(){
+        return cy.get('.main-nav')
+    }
+
+    date(){
+        return cy.get('.calendar__date')
+    }
+
+    button(){
```

Comment:
this is not very informative name of the function, it would be better to use getActiveLink or something more understandable